tests/test_document.py

```diff
@@ -188,7 +188,7 @@ def test_write_document_tv_with_validate(self):
         try:
             temp_dir = tempfile.mkdtemp(prefix='test_spdx')
             result_file = os.path.join(temp_dir, 'spdx-simple.tv')
-            with open(result_file, 'wb') as output:
+            with open(result_file, 'w') as output:
```

Comment:
Sorry, I'm not a Python expert: Will this work for Python 2, too? Because we need to stay compatible to both versions 2 and 3.

Reply:
Yes, this will work on Python 2 too. The tests pass for Python 2.7. :)

Reply:
Then I'm wondering why it ever worked for Python 3. Do you have any idea about that?

Reply:
Yes, Python 3 returns 'bytes' and not 'str' when reading/writing to/from a binary stream. So, the files need to be opened in text mode for `str` related operations.

Reply:
@sschuberth I guess the answer is in the Travis Builds - If you look at older builds that [passed](https://travis-ci.org/spdx/tools-python/jobs/243330760) the tests hadn't been added up till then (back when the builds were passing that is)

Reply:
Oh, I didn't realize @pombredanne had committed this directly, without going through a PR that runs CI. Otherwise that change should have never gotten in.